core/app/components/FormBuilder/ElementPanel/ButtonConfigurationForm.tsx

```diff
@@ -102,7 +103,11 @@ export const ButtonConfigurationForm = ({
 	const onSubmit = (values: z.infer<typeof schema>) => {
 		const index = buttonIndex === -1 ? numElements : buttonIndex;
 		update(index, {
-			order: null,
+			rank: mudder.base62.mudder(
+				elements[index - 1]?.rank ?? "",
+				elements[index + 1]?.rank ?? "",
+				1
+			)[0],
```

Comment:
This generates a rank at the end of the form elements. It's not used right now since button elements aren't ordered in the form. But maybe the should/will be eventually? In any case, I did this because I wanted to make the rank column non-nullable. Should I use a constant string here instead of the mudder calculation?